### PR TITLE
[FTX] Adding modify order by id and by clientId

### PR DESCRIPTION
**Pull request by makarid**

### CODE REVIEW

File: xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAdapters.java [hunk 1]
```diff
@@ -350,16 +352,17 @@ public static BigDecimal lendingRounding(BigDecimal value) {
...
-  public static Ticker adaptTicker(FtxResponse<FtxMarketDto> ftxMarketResp,
-                                   FtxResponse<List<FtxCandleDto>> ftxCandlesResp,
-                                   CurrencyPair currencyPair) {
+  public static Ticker adaptTicker(
+      FtxResponse<FtxMarketDto> ftxMarketResp,
+      FtxResponse<List<FtxCandleDto>> ftxCandlesResp,
+      CurrencyPair currencyPair) {
 
-    System.out.println("marketResp"  + ftxMarketResp.toString());
+    System.out.println("marketResp" + ftxMarketResp.toString());
```
Comment (earce) [inline xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAdapters.java:360]: can you change this to using the proper logger class

Comment (makarid) [reply xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAdapters.java:360]: Yes, i will change them, i don't remember writing them.

File: xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxTradeServiceRaw.java [hunk 2]
```diff
@@ -57,6 +58,38 @@ public String placeLimitOrderForSubaccount(String subaccount, LimitOrder limitOr
...
+    try {
+      return ftx.modifyOrder(
+          exchange.getExchangeSpecification().getApiKey(),
+          exchange.getNonceFactory().createValue(),
+          signatureCreator,
+          subaccount,
+          orderId,
+          payload);
+    } catch (FtxException e) {
+      throw new FtxException(e.getMessage());
```
Comment (walec51) [inline xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxTradeServiceRaw.java:73]: remove try catch

let the original exception get out

same goes for the code below

**Comment by walec51** [comment]
looks good, just one small remark

**Pull request merged**